Makefile

```diff
@@ -1,16 +1,25 @@
+.PHONY: readme
+
 readme:
 	atmos docs generate readme
 
 lint:
-	cfn-lint templates/*.yaml -I
+	cfn-lint templates/aws-cloudformation-terraform-backend.yaml
+
+validate:
+	aws cloudformation validate-template --template-body file://templates/aws-cloudformation-terraform-backend.yaml
 
+# This deploys the cloudformation template to Cloud Posse's test organization.
+# See https://github.com/cloudposse-examples/infra-demo-atmos-pro
 deploy:
-	aws cloudformation create-stack \
-		--stack-name atmos-pro-backend \
-		--template-body file://templates/atmos-pro.yaml \
-		--capabilities CAPABILITY_NAMED_IAM \
-		--parameters \
-			ParameterKey=StackName,ParameterValue=atmos-pro-backend \
-			ParameterKey=GitHubOrg,ParameterValue=cloudposse \
-			ParameterKey=GitHubRepo,ParameterValue=demo \
-			ParameterKey=CreateOIDCProvider,ParameterValue=false
+	aws cloudformation deploy \
+	  --stack-name atmos-pro-ex1 \
+	  --template-body file://templates/aws-cloudformation-terraform-backend.yaml \
+	  --capabilities CAPABILITY_NAMED_IAM \
+	  --capabilities CAPABILITY_NAMED_IAM \
+	  --no-fail-on-empty-changeset \
+	      ParameterKey=GitHubOrg,ParameterValue=cloudposse-examples \
+	      ParameterKey=CreateOIDCProvider,ParameterValue=false;
+
+delete:
+	aws cloudformation delete-stack --stack-name atmos-pro-ex1
```

Comment:
This will error for most people. Since they will not own that stack.

Reply:
same comment as above